### PR TITLE
[10.x] Add getRawQueryLog() method

### PR DESCRIPTION
**Pull request by fuwasegu**
> ⚠ NOTICE: A failed test is not about my changes

In #47507 , toRawSql() was implemented that can retrieve the complete query with values bound to the SQL output by toSql().

I thought it would be nice to have an extension method that could retrieve the complete query with bound values, similarly with DB:::getQueryLog()

I implemented it.

## Usage
Basically, the usage is the same as getQueryLog()

```php
DB::enableQueryLog();

// do something with querying

$logs = DB::getRawQueryLog();
```

In the case of getQueryLog(), we could get an associative array like this

```
[
  [
    "query" => "select * from "users" where "id" in (?, ?, ?)"
    "bindings" => [
      0 => 3
      1 => 6
      2 => 8
    ]
    "time" => 4.06
  ]
]
```

Now, we could get an associative array like this with getRawQueryLog()

```
[
  [
    "raw_query" => "select * from "users" where "id" in (3, 6, 8)"
    "time" => 4.06
  ]
]
```

### CODE REVIEW

File: src/Illuminate/Support/Facades/DB.php [hunk 1]
```diff
@@ -87,6 +87,7 @@
  * @method static void unsetTransactionManager()
  * @method static bool pretending()
  * @method static array getQueryLog()
+ * @mothod static array getRowQueryLog()
```
Comment (mpyw) [inline src/Illuminate/Support/Facades/DB.php:90]:
```suggestion
 * @mothod static array getRawQueryLog()
```

Comment (fuwasegu) [reply src/Illuminate/Support/Facades/DB.php]: thank you!

File: src/Illuminate/Support/Facades/DB.php [hunk 2]
```diff
@@ -87,6 +87,7 @@
  * @method static void unsetTransactionManager()
  * @method static bool pretending()
  * @method static array getQueryLog()
+ * @mothod static array getRawQueryLog()
```
Comment (ExeQue) [inline src/Illuminate/Support/Facades/DB.php:90]:
```suggestion
 * @method static array getRawQueryLog()
```

Comment (fuwasegu) [reply src/Illuminate/Support/Facades/DB.php:90]: thank you for your review and suggestion !

**Comment by huangdijia** [comment]
Will there be leakage of sensitive information?

**Comment by fuwasegu** [comment]
@huangdijia 
Thank you for your comment!

Certainly, there are concerns about leaking of sensitive information because this feature make it possible to dump full sql query logs which is bound all properties.
Perhaps, the one of the reasons why anyone didn't implement a function like `toRawSql()` is protecting from leaking of sensitive informatioins.
However, I think this project had decided to entrust developers to judge to use these features because toRawSql() was implemented and merged in the previous Pull Request.
Therefore, I think whether to output logs which contains sensitive informations should be based on developers judgement as well this feature.

**Comment by mitoop** [comment]
```php
  public function logQuery($query, $bindings, $time = null)
    {
        $this->totalQueryDuration += $time ?? 0.0;

        $this->event(new QueryExecuted($query, $bindings, $time, $this));

        if ($this->loggingQueries) {
            $this->queryLog[] = compact('query', 'bindings', 'time');
        }
    }
```
if this method chanage  a little, It can easily get query log by using  new `getRawQueryLog`  with the help of `DB::listen` 
```php
  public function logQuery($query, $bindings, $time = null)
    {
        $this->totalQueryDuration += $time ?? 0.0;

        // record log before dispatch event
        if ($this->loggingQueries) {
            $this->queryLog[] = compact('query', 'bindings', 'time');
        }

        $this->event(new QueryExecuted($query, $bindings, $time, $this));
    }

**Pull request merged**